packages/indexer-agent/src/agent.ts

```diff
@@ -609,40 +619,41 @@ class Agent {
       epoch,
     })
 
-    const allocationAmount = rule?.allocationAmount
+    const desiredAllocationAmount = rule?.allocationAmount
       ? BigNumber.from(rule.allocationAmount)
       : this.indexer.defaultAllocationAmount
     const desiredNumberOfAllocations = Math.max(
       1,
       rule?.parallelAllocations || 1,
     )
+    const activeAllocationAmount = activeAllocations.reduce(
+      (sum, allocation) => sum.add(allocation.allocatedTokens),
+      BigNumber.from('0'),
+    )
 
-    logger.info(`Reconcile deployment allocations`, {
-      allocationAmount: formatGRT(allocationAmount),
+    logger.trace(
```

Comment:
I like the update to include block current allocation amount and desired allocation amount.  I feel this log is one of the important ones for understanding what the indexer is doing though, think we should keep it `info` or `debug` level?

Reply:
maybe a conditional to log only if there was something to reconcile?